prometheus/prometheustest/client_test.go

```diff
@@ -298,7 +295,7 @@ func TestGetAllRequestRates(t *testing.T) {
 	}
 	mockQueryWithTime(api, `rate(istio_requests_total{source_workload_namespace="ns"}[5m])`, queryTime, &vectorQ2)
 
-	rates, err := client.GetAllRequestRates("ns", "5m", queryTime)
+	rates, _ := client.GetAllRequestRates("ns", "5m", queryTime)
```

Comment:
a bunch more 'err' removed, but again, probably OK in test code.

Reply:
I see this OK, since we are using the mocked prometheus client. The mocked client fails if a query is made and there is no mock for it. I think we are safe here, so it looks intentional to ignore the error.